metricbeat/module/php_fpm/pool/data.go

```diff
@@ -18,29 +18,94 @@
 package pool
 
 import (
-	s "github.com/elastic/beats/libbeat/common/schema"
-	c "github.com/elastic/beats/libbeat/common/schema/mapstriface"
+	"encoding/json"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
 )
 
-var (
-	schema = s.Schema{
-		"name":            c.Str("pool"),
-		"process_manager": c.Str("process manager"),
-		"slow_requests":   c.Int("slow requests"),
-		"start_time":      c.Int("start time"),
-		"start_since":     c.Int("start since"),
-		"connections": s.Object{
-			"accepted":         c.Int("accepted conn"),
-			"listen_queue_len": c.Int("listen queue len"),
-			"max_listen_queue": c.Int("max listen queue"),
-			"queued":           c.Int("listen queue"),
+type phpFpmStatus struct {
+	Name               string          `json:"pool"`
```

Comment:
What is the reason you switched from the schema library to defining the types?

Reply:
The response returned an array of objects and could not find a way to preserve it using the schema lib.  Also remapped everything in the baseEvent (https://github.com/elastic/beats/pull/8394/files#diff-6fa6396eb9530b802ab7b56fb9a8f13fR88) in order to replace the spaces with underscore.
Let me know if there is a better way to handle this.